<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Countdown</class>
 <widget class="QMainWindow" name="Countdown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Countdown</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;黑体&quot;;
background-color:rgb(255, 190, 26)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="PS">
     <attribute name="title">
      <string> Persenal State</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>158</y>
        <width>21</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 48pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="PSM">
      <property name="geometry">
       <rect>
        <x>333</x>
        <y>160</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>590</y>
        <width>21</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="PSS">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PSset">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>610</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26);
font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Set value</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PSstart">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>400</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26)</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PSpause">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>400</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26)</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Personal State</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Min">
      <property name="geometry">
       <rect>
        <x>821</x>
        <y>590</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Sec">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>590</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DB">
     <attribute name="title">
      <string> DEBATE</string>
     </attribute>
     <widget class="QLCDNumber" name="BDS1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>162</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="Sec_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>600</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
     <widget class="QPushButton" name="BDset1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>610</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26);
font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Set Positive side</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BDstart1">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>440</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26)</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Positive side</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BDpause1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26)</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="BDM1">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>162</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="Min_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>600</y>
        <width>51</width>
        <height>61</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QPushButton" name="BDpause2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>440</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26)</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>  Negative side</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BDstart2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>440</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26)</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="BDM2">
      <property name="geometry">
       <rect>
        <x>613</x>
        <y>162</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="BDS2">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>162</y>
        <width>131</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>600</y>
        <width>21</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BDset2">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>610</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 190, 26);
font: 16pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Set Negative side</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAnthor">
    <property name="title">
     <string>Anthor</string>
    </property>
    <addaction name="actionsunguiyu96_gmail_com"/>
   </widget>
   <addaction name="menuAnthor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsunguiyu96_gmail_com">
   <property name="text">
    <string>sunguiyu96@gmail.com</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
